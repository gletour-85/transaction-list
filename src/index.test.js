import ReactDom from 'react-dom';

// Mocks
// -----

jest.mock('./scripts/app', () => 'App');

const reactDomRenderSpy = jest.spyOn(ReactDom, 'render').mockImplementation();

// Tests
// -----

describe('index', () => {
  beforeEach(() => {
    require('./index');
  });

  it('renders app correctly', () => {
    expect(reactDomRenderSpy).toHaveBeenCalledTimes(1);
    expect(reactDomRenderSpy.mock.calls[0][0]).toMatchSnapshot();
  });
});
