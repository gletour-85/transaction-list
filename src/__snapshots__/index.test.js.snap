// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index renders app correctly 1`] = `
<UNDEFINED>
  <App />
</UNDEFINED>
`;
